lib/actions/collection.js

```diff
@@ -1,4 +1,4 @@
-const { StorageManager } = require('brainstem-js');
```

Comment:
Removing the dependency to the StorageManager is fantastic! I know it's outta scope so I made a follow-up issue about removing it from the defaults as well:

https://github.com/mavenlink/brainstem-redux/issues/52